packages/react/src/field/FieldLabel.tsx

```diff
@@ -4,27 +4,34 @@ import { Text } from '../text';
 
 export type FieldLabelProps = PropsWithChildren<{
```

Comment:
For consistency, I've added prop comments here while I was touching these files.